url.go

```diff
@@ -10,19 +10,25 @@ import (
 // Raw contains the complete URL.
 type URL struct {
 	version   version
-	Raw       string      `json:"raw"`
-	Protocol  string      `json:"protocol,omitempty"`
-	Host      []string    `json:"host,omitempty"`
-	Path      []string    `json:"path,omitempty"`
-	Port      string      `json:"port,omitempty"`
-	Query     interface{} `json:"query,omitempty"`
-	Hash      string      `json:"hash,omitempty"`
-	Variables []*Variable `json:"variable,omitempty" mapstructure:"variable"`
+	Raw       string           `json:"raw"`
+	Protocol  string           `json:"protocol,omitempty"`
+	Host      []string         `json:"host,omitempty"`
+	Path      []string         `json:"path,omitempty"`
+	Port      string           `json:"port,omitempty"`
+	Query     []*QueryFragment `json:"query,omitempty"`
+	Hash      string           `json:"hash,omitempty"`
+	Variables []*Variable      `json:"variable,omitempty" mapstructure:"variable"`
 }
 
 // mURL is used for marshalling/unmarshalling.
 type mURL URL
 
+type QueryFragment struct {
```

Comment:
Maybe we should name this struct `QueryParam` instead because it's the name used in the official library and also in the schema documentation :) 

Official library : https://github.com/postmanlabs/postman-collection/blob/1df7a51dcd2a498de00c9fd9810ae8ac8a35357f/lib/collection/url.js#L165

And in the documentation : 

> An array of QueryParams, which is basically the query string part of the URL, parsed into separate variables

Source : https://schema.postman.com/collection/json/v2.1.0/draft-04/collection.json